devfile.yaml

```diff
@@ -43,24 +40,17 @@ commands:
       commandLine: mvn clean install
       group:
         kind: build
-  - id: launch-eap-server
-    exec:
-      component: tools
-      workingDir: ${PROJECTS_ROOT}
-      commandLine: $JBOSS_HOME/bin/openshift-launch.sh
-      group:
-        kind: run
   - id: deploy
     exec:
-      component: tools
+      component: eap
       workingDir: ${PROJECTS_ROOT}/microprofile-quickstart/microprofile-fault-tolerance
       commandLine: mvn package wildfly:deploy && 
           echo 'The application was deployed, click on the endpoint from Workspace view to test it'
       group:
         kind: run
   - id: update
```

Comment:
can we rename this to 4-`update-existing-deployment`  so it's clear this is to push a new war, rather than something you have to do before you build?

Reply:
renamed